CONTRIBUTING.md

```diff
@@ -34,13 +34,13 @@ images might not be sufficient and a demo layer is necessary. pnorman has a serv
 
 ## Easy pickings
 
-Some [easy issues](https://github.com/gravitystorm/openstreetmap-carto/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) have been selected
+Some [easy issues](labels/good%20first%20issue) have been selected
 that are particularly suitable for new contributors to get familiar with the project's code base and the contribution process.
 
 ## Editing layers
 
 OpenStreetMap Carto uses a YAML file for defining layers, because it [works much
-better for big projects](https://github.com/gravitystorm/openstreetmap-carto/issues/711).
+better for big projects](issues/711).
```

Comment:
I would prefer leaving the longer verion of the URL including github.com/gravitystorm/openstreetmap-carto/ here and above at line 37,  so that the links will still work if this document is downloaded

Reply:
If so, in order to keep it consistent, I would also like to change the links included in the following lines:

- https://github.com/gravitystorm/openstreetmap-carto/blob/c69d8c753a5b52b6f0f9ed1084f37a950d6d7d5f/INSTALL.md?plain=1#L58
- https://github.com/gravitystorm/openstreetmap-carto/blob/c69d8c753a5b52b6f0f9ed1084f37a950d6d7d5f/README.md?plain=1#L7
- https://github.com/gravitystorm/openstreetmap-carto/blob/c69d8c753a5b52b6f0f9ed1084f37a950d6d7d5f/README.md?plain=1#L19
- https://github.com/gravitystorm/openstreetmap-carto/blob/c69d8c753a5b52b6f0f9ed1084f37a950d6d7d5f/README.md?plain=1#L23

(For GitHub's recommendation, see [Relative links](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#relative-links) section on GitHub Docs);

changing image URLs isn't my focus with this commit, but the above also applies to images..

- https://github.com/gravitystorm/openstreetmap-carto/blob/c69d8c753a5b52b6f0f9ed1084f37a950d6d7d5f/README.md?plain=1#L3

(For GitHub's recommendation, see [Images](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#images) section in GitHub Docs)

Other maintainers (like those of many other repositories) may follow GitHub's recommendations (maybe simply to make it easy to search file names) and therefore be against changing these links.

Reply:
Good point, it appears the shorter version is more widely supported. I’ll remove my request for changes.